### PR TITLE
Remove obvious comments from tests

### PR DESCRIPTION
**Pull request by dshil**
Resolves: #66

### CODE REVIEW

**Comment by dshil** [comment]
@eugulixes I've removed comments using the following logic:
- The docstring of the helper method is well documented.
- Test doesn't have the entire logic, that's why it's enough to see the docstring of the helper method and understand what the hell going on there.

File: gits/test/capabilities_test.py [hunk 1]
```diff
@@ -300,18 +255,13 @@ def test_cap_cup(self):
         """
         term = self._terminal
 
-        # The cursor is at the left-most position.
-        # Note that the y and x values start from 1.
```
Comment (eugulixes) [inline gits/test/capabilities_test.py]: I know that there's a corresponding comment in `_check_cap_cup`, but still it's quite useful.

**Comment by eugulixes** [comment]
@dshil, I think there's [one more](https://github.com/tolstoyevsky/gits/pull/67/files#diff-e0594523c726dfb1c6806dc4c24e3373R410) redundant comment.

**Comment by dshil** [comment]
ping @eugulixes, could your make a review for this PR?

**Comment by eugulixes** [comment]
@dshil, you didn't write anything about [one more](https://github.com/tolstoyevsky/gits/pull/67/files#diff-e0594523c726dfb1c6806dc4c24e3373R410) redundant comment. Probably I'm mistaken and it's not redundant. Let me know=)

**Review by eugulixes — APPROVED** [review]
Great job!

**Pull request merged**